backend/drive/drive.go

```diff
@@ -1078,13 +1083,19 @@ func newFs(name, path string, m configmap.Mapper) (*Fs, error) {
 
 	// Create a new authorized Drive client.
 	f.client = oAuthClient
-	f.svc, err = drive.New(f.client)
+
+	f.svc, err = drive.NewService(ctx, option.WithHTTPClient(f.client))
 	if err != nil {
 		return nil, errors.Wrap(err, "couldn't create Drive client")
 	}
 
+	f.activitySvc, err = driveactivity.NewService(ctx, option.WithHTTPClient(f.client))
+	if err != nil {
+		return nil, errors.Wrap(err, "couldn't create Drive Activity client")
```

Comment:
This is going to cause a backwards incompatibility problem as this will fail for anyone who doesn't have the correct scope in their oauth token.

So we need to fail soft here, maybe with a message to do `rclone config reconnect backend:`

It might be nice to delay opening this until the ChangeNotify service is used also as people who are just doing syncing don't care about changes.

Reply:
I think that's what happens anyway. I don't think it does any API call till the first request. I assume the tests would have failed otherwise?